fv/containers/containers.go

```diff
@@ -337,25 +337,36 @@ func RunEtcd() *Container {
 		"--listen-client-urls", "http://0.0.0.0:2379")
 }
 
-func RunFelix(etcdIP string) *Container {
+func RunFelix(etcdIP string, options TopologyOptions) *Container {
 	log.Info("Starting felix")
 	return Run("felix",
 		RunOpts{AutoRemove: true},
 		"--privileged",
 		"-e", "CALICO_DATASTORE_TYPE=etcdv3",
 		"-e", "CALICO_ETCD_ENDPOINTS=http://"+etcdIP+":2379",
-		"-e", "FELIX_LOGSEVERITYSCREEN=debug",
+		"-e", "FELIX_LOGSEVERITYSCREEN="+options.felixLogSeverity(),
 		"-e", "FELIX_DATASTORETYPE=etcdv3",
 		"-e", "FELIX_PROMETHEUSMETRICSENABLED=true",
 		"-e", "FELIX_USAGEREPORTINGENABLED=false",
 		"-e", "FELIX_IPV6SUPPORT=false",
 		"calico/felix:latest")
 }
 
+type TopologyOptions struct {
+	FelixLogSeverity string
+}
+
+func (o TopologyOptions) felixLogSeverity() string {
+	if o.FelixLogSeverity == "" {
+		return "debug"
+	}
+	return o.FelixLogSeverity
+}
```

Comment:
If we imagine TopologyOptions having more fields with default values, I think I'd prefer a single constructor func that establishes all the defaults, rather than a method like this for each field.